test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/TestSuitInitializer.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+	/* Responsible for all initilisation before actual test startup -
+	* Creation of Azure resources - Eventhubs and Storage Queues
+	* Function App startup
+	*/
+	public class TestSuitInitializer
+	{
+		private readonly ILogger _logger = TestLogger.GetTestLogger();
+
+		public void InitializeTestSuit(Language language, BrokerType brokerType)
+		{
+			CreateAzureResources(language, brokerType);
+			Task.WaitAll(StartupApplicationAsync(language, brokerType));
+		}
```

Comment:
You must use the inbuild formatter. Necessary for clean code. There are various good practices in formatting which these formatter fixes.. We can check together and see if we can make it work. There are many formatting errors accross the files.

Reply:
Formatted using vs extension.